articles/language/type-model.md

```diff
@@ -338,19 +338,20 @@ with the same result type and an input type that is compatible with `'A`.
 That is, given the following definitions:
 
 ```qsharp
-operation Invertible (qs : Qubit[]) : Unit 
+operation Invert(qs : Qubit[]) : Unit 
 is Adj {...} 
-operation Unitary (qs : Qubit[]) : Unit 
+
+operation Unitary(qs : Qubit[]) : Unit 
```

Comment:
On a different page, I saw that you changed `qs` -> `qubits` in this same context.

Reply:
I have tried to regularized all of the changes to make `qubit` the default singular qubit variable name, and `qubits` if it is an array.